### DISCUSSION TITLE
Adding Drag (and drop) Tailwind Calendar

### DISCUSSION THREAD

**Discussion by mrm2234** [tailwindlabs/tailwindcss#11417] [Help]
Does anyone know how to enable event creation on drag within a calendar like how google calendar does this? For example, you can drag on the day view and it'll create an event for the space in which you dragged your cursor. 

Similarly, does anyone know how I could drag and drop an event on to the calendar?

**Comment by wongjn** [comment]
You could look at the [Drag and Drop API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API). However, this question does not seem related to Tailwind – consider asking in a discussion area more related to your topic.
